pxtlib/sprite-editor/spriteEditor.ts

```diff
@@ -223,6 +223,24 @@ namespace pxtsprite {
             this.edit = this.newEdit(this.color);
         }
 
+        initializeUndoRedo(undoStack: Bitmap[], redoStack: Bitmap[]) {
+            if (undoStack) {
+                this.undoStack = undoStack;
+            }
+            if (redoStack) {
+                this.redoStack = redoStack;
+            }
+            this.updateUndoRedo();
+        }
+
+        getUndoStack() {
+            return this.undoStack;
```

Comment:
Should probably do a `.slice(0)` here; same with the redo stack. Will this get duplicated if I copy/paste a block?

Reply:
It doesn't get get duplicated when duplicating the block, though that would be a real nice feature; I haven't interacted with blockly duplication stuff yet though so I'll have to look into how that would work

Reply:
I just checked; doesn't look possible. Copy/paste works with the XML and we don't want to put the stack in the XML